backend/src/main/java/aiin/backend/domains/member/domain/Logout.java

```diff
@@ -0,0 +1,19 @@
+package aiin.backend.domains.member.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.redis.core.RedisHash;
+import org.springframework.data.redis.core.index.Indexed;
+
+@Getter
+@AllArgsConstructor
+@RedisHash(value = "blacklist", timeToLive = 60 * 60)
+public class Logout {
```

Comment:
access token을 블랙 리스트에 담으셨군요. accessToken을 key로 사용하고 refresh token을 value로 설정한 객체를 블랙리스트에 담아서 TTL을 길게 설정하는 방안도 고려해보면 좋을 듯 싶습니다! 
```suggestion
public class Logout {
    @Id
    private String accessToken;
    
    private String refreshToken;
```